java/test/jmri/jmrix/AbstractConnectionConfigTestBase.java

```diff
@@ -37,6 +37,7 @@ public void testName(){
     public void testGetInfo(){
         Assume.assumeNotNull("adapter set", cc.getAdapter());
         Assert.assertNotNull("has info", cc.getInfo());
+        jmri.util.JUnitAppender.suppressErrorMessage("No usable ports returned");
```

Comment:
Why is this needed? I think it would be better to move this line to the classes that needs it, so that they override this method. Otherwise some tests that should fail due to this will not fail.

If I add a new connection config there a usable port is expected, but for whatever reason that port doesn't exists, I would like to get this error.

Reply:
Whoops, was meaning to explore this with the 2 local failures I'm consistently getting with this test,

jmrix.can.adapters.lawicell.canusb.serialdriver.ConnectionConfigTest
jmrix.dccpp.serial.ConnectionConfigTest

though as they're not causing CI failures not high priority so locally suppressed so I could run a coverage report on jmrix !

Have removed for now, Steve.